<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frame_widget</class>
 <widget class="QWidget" name="frame_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_frame_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
border: 1px solid white;
background-color: black;</string>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Id : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_width">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_height">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height : 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete_frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_move_up">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_move_down">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
